x/genutil/client/cli/collect.go

```diff
@@ -18,7 +18,7 @@ import (
 const flagGenTxDir = "gentx-dir"
 
 // CollectGenTxsCmd - return the cobra command to collect genesis transactions
-func CollectGenTxsCmd(genBalIterator types.GenesisBalancesIterator, defaultNodeHome string) *cobra.Command {
+func CollectGenTxsCmd(genBalIterator types.GenesisBalancesIterator, defaultNodeHome string, m genutil.AppModuleBasic) *cobra.Command {
```

Comment:
This function should instead take an argument of type `GenTxValidator` (whatever that type is). There isn't a need to pass the entire `AppModuleBasic` IMO

Reply:
ok, done